apps/shared/avifutil.h

```diff
@@ -20,7 +20,7 @@
 #define AVIF_FMT_ZU "%zu"
 #endif
 
-void avifImageDump(avifImage * avif, uint32_t gridX, uint32_t gridY);
+void avifImageDump(avifImage * avif, uint32_t gridCols, uint32_t gridRows);
```

Comment:
The caller (avifenc.c) calls avifImageDump() as follows:

`        avifImageDump(avif, 0, 0);`

and

`    avifImageDump(gridCells ? gridCells[0] : image, gridDims[0], gridDims[1]);`

The second call makes it clear that these two parameters are gridCols and gridRows.